assets/js/data/index.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Wrapper for the wcSettings global, which sets defaults if data is missing.
+ *
+ * Only settings used by blocks are defined here. Component settings are left out.
+ */
+export const currency = wcSettings.currency || {
+	code: 'USD',
+	precision: 2,
+	symbol: '$',
+	position: 'left',
+	decimal_separator: '.',
+	thousand_separator: ',',
+	price_format: '%1$s%2$s',
+};
```

Comment:
Yup, me likey 👍 

So one thing we may also want to consider is add an alias to the webpack config so we can avoid doing relative imports everytime we need to use data. We could alias `wcdata` for instance to point to this file and then only have to do `import { currency } from 'wcdata'` in our modules.